xAPI-Communication.md

```diff
@@ -189,6 +191,12 @@ the xAPI as described above due to only supporting "GET" and "POST", and not all
 The following describes alternate syntax to use only when unable to use the usual syntax for specific calls due to the 
 restrictions mentioned above. This alternate syntax can also be used to GET Statements due to limits on query string length.  
 
+All requirements for this environment are necessary only if supporting versions of browsers requiring this type of support.
```

Comment:
Repeated from above: This is not true. LRSs *MUST* support this and clients *MAY* use this even when not using IE9 or lower.